src/Components/Wrapper/Wrapper.tsx

```diff
@@ -189,6 +189,17 @@ const Wrapper = (props: PropsWithChildren<{}>) => {
     document.documentElement.removeAttribute('dir');
   }, [locale]);
 
+  useEffect(() => {
+    if (!referrerData) return;
+    const defaultLang = getReferrer('defaultLanguage', 'en-us');
+    const theme = getReferrer('theme', 'default');
+
+    const isCobrand = theme !== 'default';
```

Comment:
I'm wondering about this `isCobrand` check, I think we might want to remove it. My thinking is, in the hispanic federation case (referrer code is hfed), I think this condition would prevent their co-brand from defaulting to Spanish like we want. I think maybe the term "cobrand" can be interpreted multiple ways but in my mind it's any referrer code we use.